### PR TITLE
test case for CREATE

### PR DESCRIPTION
**Pull request by jaewhyun**
wrote a test case for CREATE API  call

### CODE REVIEW

**Comment by jaewhyun** [comment]
@annelai @gau820827 @katydid98 Added CREATE test case in test_service.py. Please review!

File: test_service.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        resp = self.app.post('/recommendations', data=data, content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+        # Make sure location header is set
+        location = resp.headers.get('Location', None)
+        self.assertTrue(location != None)
+        # Check the data is correct
+        new_json = json.loads(resp.data)
+        self.assertEqual(new_json['product_id'], 6)
+        # check that count has gone up and includes 4
+        resp = self.app.get('/recommendations')
```
Comment (annelai) [inline test_service.py]: I think you can replace line 67-70 by just one line like this:
`new_recommendation_count = self.get_recommendation_count()`
reuse the code and make it more clear to read!

File: test_service.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Test cases can be run with:
+  nosetests
+  coverage report -m
+"""
+
+import logging
+import unittest
+import json
+from flask_api import status    # HTTP Status Codes
+from models import DataValidationError
```
Comment (annelai) [inline test_service.py]: DataValidationError is imported but not used?

File: test_service.py (hunk 1 above)
Comment (jaewhyun) [reply test_service.py]: I tried but nosetest didn't pass because I needed the *data* that I got back for *assertIn*

**Comment by jaewhyun** [comment]
@annelai @gau820827 I fixed the file. Let me know what you guys think. I couldn't use Anne's suggestion cause it threw nosetest errors. If it can be improved, I'll improve it later

**Pull request merged**